src/main/java/com/schibsted/spt/data/jslt/impl/PipeOperator.java

```diff
@@ -0,0 +1,39 @@
+
+// Copyright 2018 Schibsted Marketplaces Products & Technology As
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.schibsted.spt.data.jslt.impl;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+import com.schibsted.spt.data.jslt.JsltException;
+
+public class PipeOperator extends AbstractOperator {
+
+  public PipeOperator(ExpressionNode left, ExpressionNode right,
+                      Location location) {
+    super(left, right, "|", location);
+  }
```

Comment:
This object is missing `optimize()`. I can't think of any optimization opportunities for the pipe operator, but we are now losing out on any optimization that could happen inside `left`  and  `right`

Reply:
I think since `PipeOperator` inherits from `AbstractOperator` the `AbstractOperator.optimize()` handles this. (by calling `left.optimize()` and `right.optimize` see [AbstractOperator.java](https://github.com/schibsted/jslt/blob/7274a94940500ba758a4eb1351d7036b933fc14f/src/main/java/com/schibsted/spt/data/jslt/impl/AbstractOperator.java#L50-L60))

Reply:
You're right. My bad. Sorry. Better to take the method out. (I keep forgetting about `AbstractOperator`.)

Reply:
Done!